aws_lambda_powertools/utilities/parser/models/s3.py

```diff
@@ -11,88 +11,319 @@
 
 class S3EventRecordGlacierRestoreEventData(BaseModel):
     lifecycleRestorationExpiryTime: datetime
```

Comment:
Can you add a description and an example here? Please let me know if you need help writing them.